### ISSUE TITLE
Translating in loops within template doesn't work

### ISSUE THREAD

**Issue by mahmoudajawad** [robisim74/qwik-speak#6]
I'm attempting to dynamically load values and then translate them in a loop within template as to offer a select menu with dynamic options. Take following example:
```jsonc
// i18n/LANG/app.json
{
  "termA": "Value for term A",
  "termB": "Value for term B",
  "termC": "Value for term C",
```

```tsx
// componens.tsx
const TERMS = ["A", "B", "C"];

return (
  <select>
    {TERMS.map((term) => <option>{translate(`term${term`)}</option>)}
  </select>
);
```

The end results show up the terms without being translated. I tested this scenario in qwik-speak 0.0.5-0.0.8 and it was consistent.

### Workaround
As to workaround this problem, I found translating the values outside of template, in a loop, does work such as:
```tsx
// const TERMS...
const VALUES = TERMS.map((term) => translate(`term${term}`));

// ...

  <select>
    {VALUES.map((value) => <option>{value}</option>)}
  </select>
```

**Comment by robisim74** [comment]
That's weird. It seems to work: https://stackblitz.com/edit/qwik-speak-8xx6qd?file=src%2Fapp%2Froutes%2F[...lang]%2Findex.tsx

Then the `translate` function also accept an array of keys, as in example.

**Comment by mahmoudajawad** [comment]
Yes it is. Does the stack-blitz use the published version (0.0.8) or are there any changes that are included but not yet published? I'll try again with a clean project, although the issue showed-up in two almost-clean projects, but there's still a chance I missed something up.
I'll update you.

**Comment by robisim74** [comment]
@mahmoudajawad Did you solve it?
